dbms/src/WindowFunctions/tests/gtest_last_value.cpp

```diff
@@ -107,7 +106,6 @@ class LastValue : public DB::tests::ExecutorTest
     template <typename FloatType>
     void testFloat()
     {
-        // TODO test with bounded_type_frame
```

Comment:
Why remove the comment? Is bounded_type_frame tested here?

Reply:
> Why remove the comment? Is bounded_type_frame tested here?

Yes, the unbounded and offset type are already tested here.